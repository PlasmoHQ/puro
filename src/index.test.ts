import { expect, test } from "@jest/globals"
import { execFileSync } from "child_process"
import { join } from "path"
import { cwd, execPath } from "process"

import { life } from "~index"

const indexScript = join(cwd(), "dist", "index.js")

test("life is good", async () => {
  expect(life).toBe(42)
})

test("snapshot corrects", async () => {
  const output = execFileSync(execPath, [indexScript]).toString("utf-8")
  expect(output).toMatchSnapshot()
})
