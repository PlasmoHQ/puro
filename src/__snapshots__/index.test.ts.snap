// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot corrects 1`] = `
"HELLO WORLD 42
"
`;
